provisioning/plugins/corim-psa-decoder/ccaplatformconfigid.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	structpb "google.golang.org/protobuf/types/known/structpb"
+)
+
+type CCAPlatformConfigID struct {
+	Label string
+	Value []byte
+}
+
+func (o *CCAPlatformConfigID) FromMeasurement(m comid.Measurement) error {
+	id, err := m.Key.GetCCAPlatformConfigID()
```

Comment:
curiosity, if mkey is not present in the comid, will `m.Key` be `nil`?

Reply:
Yes correct! I think, we check it first and then land in this function!

Reply:
In the top level processing of CoMID